README.md

```diff
@@ -1,6 +1,6 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains Modest's portfolio and SPS projects.
```

Comment:
It's always a good practice to update README for changes. Good meaningful description